core/client/src/main/java/alluxio/client/block/AlluxioBlockStore.java

```diff
@@ -13,6 +13,7 @@
 
 import alluxio.Constants;
 import alluxio.client.ClientContext;
+import alluxio.client.file.options.InStreamOptions;
```

Comment:
Hmm okay, it seems like we already using it this way, I've opened a ticket to improve this: https://alluxio.atlassian.net/browse/ALLUXIO-2423